R/dev-help.R

```diff
@@ -26,6 +26,14 @@ dev_help <- function(topic,
                      dev_packages = NULL,
                      stage = "render",
                      type = getOption("help_type")) {
+
+  if (length(dev_packages()) == 0) {
+    cli::cli_abort(c(
+      "No in-development packages found.",
```

Comment:
A bit more context for this error would be nice. Perhaps with a contextual info bullet? Do we allow this structure with pre and post info bullets? This is consistent with the new dplyr chained error. The idea is that contextual information always precede the error message. Advices always trail.

```
Errror in `dev_help()`:
i In the devtools shim for `?`.
! An in-development package must be loaded in the session.
i Do you need to run `pkgload::load_all()`?
```

I also tweaked the main message to use "must" phrasing.

Reply:
I just added a bit more context to the error itself: Can't find development documentation because no in-development packages loaded. I think that should be adequate for the places where you'll hit this.